pkg/scheduler/core/extender_test.go

```diff
@@ -198,7 +198,7 @@ func (f *FakeExtender) selectVictimsOnNodeByExtender(
 	// and get cached node info by given node name.
 	nodeInfoCopy := f.cachedNodeNameToInfo[node.GetName()].Clone()
 
-	potentialVictims := util.SortableList{CompFunc: util.MoreImportantPod}
+	potentialVictims := podinfo.SortableList{CompFunc: podinfo.MoreImportantPod}
```

Comment:
It feels like SortableList shouldn't belong to the podinfo package.

Reply:
@alculquicondor Can you suggest a package to move it into?

Reply:
what about `internal/collections` or more specific `internal/sortablelist`